concourse/pipelines/artifact-releaser-test.jsonnet

```diff
@@ -445,7 +445,7 @@ local pkggroup = {
                    owner: 'GoogleCloudPlatform',
                    repository: package,
                    access_token: '((github-token.token))',
-                   tag_filter: '([0-9]+.*)',
+                   tag_filter: '([0-9]+\\.[0-9]+)',
```

Comment:
is this expression change meant to match both `certgen.x86_64.x86_64` and `certgen.x86_64`?

Reply:
This is for matching versions, so it won't be matching the file names. According to the errors in concourse right now, it is getting the `20180522a` tag for `compute-image-tools`. The extra `a` messes things up.